hw/ip/aes/data/aes_testplan.hjson

```diff
@@ -75,6 +75,21 @@
             - Write unsupported configurations (Key length and mode are 1 hot, what happens if more than one bit is set.)'''
       milestone: V2
       tests: []
+    }
+	{
```

Comment:
nit: brackets and desc are misaligned. Otherwise, LGTM.

Reply:
solved - thank you